MockKitUI/View Controllers/MockServiceTableViewController.swift

```diff
@@ -357,35 +339,23 @@ fileprivate extension UIAlertController {
             style: .destructive,
             handler: { _ in
                 handler()
-        }
-        ))
+        }))
 
         let cancel = NSLocalizedString("Cancel", comment: "The title of the cancel action in an action sheet")
         addAction(UIAlertAction(title: cancel, style: .cancel, handler: nil))
     }
-    
-    convenience init(versionCheckHandler handler: @escaping (VersionUpdate) -> Void) {
-        self.init(title: "Version Check Response",
-                  message: "How should the simulator respond to a version check?",
-                  preferredStyle: .actionSheet
-        )
-        for versionUpdate in VersionUpdate.allCases {
-            addAction(UIAlertAction(title: versionUpdate.localizedDescription, style: versionUpdate.actionStyle, handler: { _ in handler(versionUpdate) }))
-        }
-        addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))
-    }
 }
 
 fileprivate extension VersionUpdate {
```

Comment:
This should probably be elsewhere if you just removed all of the other update related code from the `MockServiceTableViewController`.